lib/logger.js

```diff
@@ -1,7 +1,57 @@
 'use babel'
 
+import _ from 'lodash'
+
 export default class Logger {
-  error (/* statusCode, result, builder */) {}
-  warning (/* message */) {}
-  info (/* message */) {}
+  constructor () {
+    this.messages = []
+    this._group = null
+    const callback = () => { this.showFilteredMessages() }
+    atom.config.onDidChange('latex.loggingLevel', callback)
```

Comment:
I agree! 894052f6816d99bea465bcc601299b974b694459 removes it without changing any logic.